src/plugins/intel_gna/src/frontend/model_quantizer.hpp

```diff
@@ -148,7 +148,7 @@ class ModelQuantizer {
                 inf_loop_count++;
             } else {
                 if (inf_loop_count > 0 &&
```

Comment:
```suggestion
                if ((inf_loop_count > 0) &&
```

Reply:
Thanks @andrei-kochin . We decided that we need to check this more carefully and revert first.